### PR TITLE
Add benchmark as instance metadata for AWS/GCE.

### PR DESCRIPTION
**Pull request by cmccoy**
Tested by running netperf on AWS, GCE; verified that instance metadata was set
correctly.

### CODE REVIEW

**Comment by cmccoy** [comment]
test this please.

**Comment by cmccoy** [comment]
@ehankland: would you mind taking a look?

File: perfkitbenchmarker/aws/util.py [hunk 1]
```diff
@@ -21,23 +21,38 @@
...
+  By default, resources are tagged with "owner" and "perfkitbenchmarker-run"
+  key-value
+  pairs.
+
+  Args:
+    resource_id: An extant AWS resource to operate on.
+    region: The AWS region 'resource_id' was created in.
+  """
+  tags = {'owner': FLAGS.owner, 'perfkitbenchmarker-run': FLAGS.run_uri}
+  return AddTags(resource_id, region, **tags)
```
Comment (ehankland) [inline perfkitbenchmarker/aws/util.py]: nit: no need for the return

**Comment by ehankland** [comment]
LGTM

**Comment by cmccoy** [comment]
test this please.

**Pull request merged**